### ISSUE TITLE
Cannot read properties of undefined (reading 'visitCount')

### ISSUE THREAD

**Issue by thealves** [rowanwins/polygon-splitter#9]
Cannot read properties of undefined (reading 'visitCount') while trying to split the polygon below:

Polygon:
```
{
    "type": "Polygon",
    "coordinates": [
        [
            [
                32.80705153455757,
                -90.75240431718575
            ],
            [
                32.806936459339084,
                -90.75505954503734
            ],
            [
                32.813606246566806,
                -90.75512391896869
            ],
            [
                32.81407491549404,
                -90.74958776087286
            ],
            [
                32.81058948892064,
                -90.74962025902433
            ],
            [
                32.8105237854936,
                -90.75242021385212
            ],
            [
                32.80705153455757,
                -90.75240431718575
            ]
        ]
    ]
}
```

Line:

```
{
    "type": "LineString",
    "coordinates": [
        [
            32.80657591603978,
            -90.7493517231246
        ],
        [
            32.80920784849386,
            -90.75117565117942
        ],
        [
            32.81418306951084,
            -90.75218417610382
        ]
    ]
}
```

Also im using Turf @turf/line-intersect to identify if the line really intersects before sending to polygon-splitter, and on the Turf validation it is ok returning:

```
{
    "type": "FeatureCollection",
    "features": [
        {
            "type": "Feature",
            "properties": {},
            "geometry": {
                "type": "Point",
                "coordinates": [
                    32.81386064636804,
                    -90.75211881784645
                ]
            }
        },
        {
            "type": "Feature",
            "properties": {},
            "geometry": {
                "type": "Point",
                "coordinates": [
                    32.810546622037506,
                    -90.75144703339186
                ]
            }
        }
    ]
}
```

Can you check what is going on? Because it intersects, the turf validation is returning values.

<img width="786" src="https://user-images.githubusercontent.com/55852987/159709226-eac2bf21-9144-40bf-bc6c-6132a41f54a5.png">

**Comment by rowanwins** [comment]
Thanks for the nice clear bug report @thealves - I'll do some digging and get back to you

**Comment by rowanwins** [comment]
Fixed in 0.0.8 which I've just released - bit of silly bug that snuck past somehow.
